treeherder/config/settings.py

```diff
@@ -299,9 +299,9 @@ def server_supports_tls(url):
 CELERYD_TASK_TIME_LIMIT = CELERYD_TASK_SOFT_TIME_LIMIT + 30
 
 CELERYBEAT_SCHEDULE = {
-    'fetch-push-logs-every-minute': {
+    'fetch-push-logs-every-5-minutes': {
```

Comment:
Do you want to add a comment on how this is just a failsafe? Other than that :shipit: